pkg/runner/run_context.go

```diff
@@ -462,17 +474,45 @@ func (rc *RunContext) getGithubContext() *githubContext {
 	if runNumber == "" {
 		runNumber = "1"
 	}
+	actionPath := rc.Config.Env["GITHUB_ACTION_PATH"]
+	actionRef := rc.Config.Env["RUNNER_ACTION_REF"]
+	actionRepository := rc.Config.Env["RUNNER_ACTION_REPOSITORY"]
+	githubPath := rc.Config.Env["GITHUB_PATH"]
+	runnerUser := rc.Config.Env["RUNNER_USER"]
+	if runnerUser == "" {
+		runnerUser = "runner"
+	}
```

Comment:
Why?

Reply:
This matches GitHub's implementation.

Reply:
But we shouldn't hardcode user especially since in `micro` and `medium` images this is not `runner`.

Reply:
I'm open to suggestions (happy to chatter on gittter if that's easier).

Do you have an example workflow with either of those?

Reply:
Replied to you on Gitter, @jsoref